src/layouts/default.astro

```diff
@@ -182,6 +183,7 @@ const canonical = new URL(path, Astro.site);
   <body>
     <Header />
     <slot />
+    <ContactFooter />
```

Comment:
If we add it here, we will be able to see that footer as well as the form at the top of the page on contact.

that being said, I think having the contact form itself to re-use with both locations would be handy 🙌